src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -342,6 +342,17 @@ protected final String getSuccessfulDestination() {
         return triggers.toMap();
     }
 
+    public List<TriggerDescriptor> getTriggerDescriptors() {
+        List<TriggerDescriptor> result = new ArrayList<TriggerDescriptor>();
+        for (TriggerDescriptor d: Trigger.for_(this)) {
+            if (d instanceof TimerTrigger.DescriptorImpl) {
+                continue;
```

Comment:
I am assuming that *core* will add `DescriptorVisibilityFilter` support to `Trigger.for_(Item)` in which case this is just a second level filtering